tsconfig.app.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "@vue/tsconfig/tsconfig.dom.json",
-  "include": ["env.d.ts", "src/**/*", "src/**/*.vue"],
+  "include": ["env.d.ts", "src/**/*", "src/**/*.vue", "src/i18n/**/*.json"],
```

Comment:
What is this for?

Reply:
This is telling typescript to be aware of the files, Vite inherits this and loads them automatically (when imported)